projects/notary/project.yaml

```diff
@@ -0,0 +1,10 @@
+homepage: "https://github.com/notaryproject/notary"
+main_repo: "https://github.com/notaryproject/notary"
```

Comment:
Can we change the `main_repo` to an org address `https://github.com/notaryproject`? As we are developing Notation v2 (Notation), the original Notary v1 `https://github.com/notaryproject/notary` is not in active development and iteration.

Reply:
Done